Edward-K1/MyDiaryWithReact
# #162267199 create landing page

#### What does this PR do?
It adds a landing page to the project
#### Description of Task to be completed?

- Create a landing page
- Link to user signup and login pages

#### How should this be manually tested?

- Run `yarn install` to install dependencies and `yarn start` to start the server
- The landing page should now be displayed

#### What are the relevant pivotal tracker stories?
[#162267199](https://www.pivotaltracker.com/story/show/162267199)
